samples/bluetooth/iso_time_sync/README.rst

```diff
@@ -0,0 +1,364 @@
+.. _bluetooth-isochronous-time-synchronization:
+
+Bluetooth: ISO time synchronization
+###################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+The Bluetooth® ISO time synchronization sample showcases time-synchronized processing of isochronous data.
+
+The sample demonstrates:
+
+ * How to send isochronous data on multiple streams in such a way that all receiving devices can process the data at the same time.
+ * How receiver(s) of the isochronous data can process received data synchronously.
+ * How to provide the isochronous data to the controller right before it is sent on air.
+ * How to achieve this for either broadcast (BIS) or over connections (CIS).
+
+This sample demonstrates this by configuring a single device as a transmitter of its **BTN1** state.
+The transmitting and receiving device(s) toggle **LED1** synchronously with microsecond accuracy.
+**LED2** is toggled upon sending or receiving the data to allow measurement of the minimal the end-to-end latency.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+The sample also supports other development kits running SoftDevice Controller variants that support isochronous channels (see :ref:`SoftDevice Controller feature support <softdevice_controller>`).
+In order to demonstrate time synchronization between transmitter and receiver, two development kits are needed.
+To demonstrate time synchronization between multiple receivers, at least three development kits are needed.
+You can use any of the development kits mentioned above and mix different development kits.
+
+Additionally, the sample requires a connection to a computer with a serial terminal for each of the development kits.
+
+To be able to oberserve the microsecond accurate toggling of the LEDs, a logic analyzer or an oscilloscope is needed.
+
+Overview
+********
+
+This section first presents a quick overview of isochronous channels.
+The latter part of this section presents the structure of this sample application
+
+Isochronous channels
+====================
+
+The Bluetooth LE Isochronous Channels feature is a way of using Bluetooth LE to transfer timebounded data between devices.
+It provides a mechanism that makes sure that multiple sink devices, receiving data from the same source, will render it at the same time.
+Data has a time-limited validity period, at the end of which it is said to expire.
+Expired data which has not yet been transmitted, will be discarded.
+This means that receiver devices only ever receive data which is valid with respect to rules regarding its age and acceptable latency.
+
+The key concepts of isochronous channels are the following:
+
+Topology
+    Isochronous channels can either be used for one-to-many (Broadcast) or one-to-one (Connected).
+    When using connected isochronous streams, data may be time-synchronized between multiple connected devices.
```

Comment:
maybe you could add a second sentence about what you mean with time-synchronized.